src/showcase/index.md

```diff
@@ -42,29 +46,51 @@ toc: false
         {% for case in site.data.showcases %}
             {% capture modulo3 %}{{ forloop.index0 | modulo:3 }}{% endcapture %}
             {% if modulo3 == '0' %}
-                <div class="showcase__apps__cards text-left card-deck">
+                <div class="showcase__apps__cards text-left card-deck flex-column flex-lg-row">
             {% endif %}
                     <div class="card">
+                        <div class="card-header">
+                            {% if case.logo_src %}
+                                {% asset '{{ case.logo_src }}' class='showcase__apps__logo' %}
+                            {% endif %}
+                        </div>
                         <div class="card-body">
                             <h3>{{ case.name }}</h3>
                             <p>{{ case.description }}</p>
                         </div>
                         <div class="card-footer">
-                            <a class="btn btn-link" href="{{ case.learn_more_link }}">Learn more</a>
+                            {% if case.learn_more_link %}<a href="{{ case.learn_more_link }}">Learn more</a>{% endif %}
+                            {% if case.play_store_link or case.app_store_link %}
+                                <span class="dropdown">
+                                    <a href="#" class="dropdown-toggle" id="dropdownMenuButton{{ forloop.index }}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                        Download
+                                    </a>
+                                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton{{ forloop.index }}">
+                                        {% if case.play_store_link %}<a class="dropdown-item" href="{{ case.play_store_link }}" target="_blank">Android</a>{% endif %}
+                                        {% if case.app_store_link %}<a class="dropdown-item" href="{{ case.app_store_link }}" target="_blank">iOS</a>{% endif %}
+                                    </div>
+                                </span>
+                            {% endif %}
```

Comment:
Regarding the "Download" feature: could we ...

- Right-justify it in the card?
- Replace the "Download" text with a download icon?

Actually, another idea would be to replace the dropdown with platform specific icon(s), floated to the right. WDYT?